### ISSUE TITLE
Old version of openapi-to-postmanv2 causing postman collection server (variable) issue?

### ISSUE THREAD

**Issue by JaredAAT** [apideck-libraries/portman#437]
It might also be related to https://github.com/apideck-libraries/portman/issues/408 too.

You're currently using openapi-to-postmanv2 version 2.14.1 and I think it's malforming servers.  

Consider this openAPI.json:

```
{
  "openapi": "3.0.3",
  "paths": {
    "/node/{id}": {
      "get": {
        ...
      }
    }
  }
  "servers": [
    {
      "url": "{url}/content",
      "description": "The endpoint",
      "variables": {
        "url": {
          "default": "https://api.example.com",
          "description": "the API URL"
        }
      }
    }
  ],
}
```

Your postman collection is generating this when running: `npx @apideck/portman -l ./dev/openAPI.json -n true`

```
{
  "_": {
    "postman_id": "7fd081ba-7f6c-4e8e-af7d-b97866a10c5d"
  },
  "variable": [
    {
      "type": "string",
      "value": "https://api.example.com",
      "key": "baseUrl"
    },
    {
      "type": "string",
      "value": "{{url}}/content",
      "key": "baseUrl"
    }
  ],
}
```

At which point it errors with:

```
Content API

❏ Content API
↳ Get a Node document. Returns in an array.
  GET {{url}}/content/node/non fugiat magna?status=1&loadRef=1 [errored]
     getaddrinfo ENOTFOUND {{url}}
```

Using openapi-to-postmanv2 version 4.4.1, it'll generate a postman like:

```
{
  "variable": [
        {
            "description": {
                "content": "the API URL",
                "type": "text/plain"
            },
            "type": "any",
            "value": "https://uat-api.aat.org.uk",
            "key": "url"
        },
        {
            "type": "string",
            "value": "{{url}}/content",
            "key": "baseUrl"
        }
    ],
}
```

Which as you can see, takes into account {{url}}

**Comment by thim81** [comment]
Hi @JaredAAT ,

We are indeed a bit behind with the version of the openapi-to-postman library.
We have plans to migrate to the latest version as part of bumping support for OAS 3.1 https://github.com/apideck-libraries/portman/issues/314

The holidays are coming up, so that might give so more time to get the library upgraded and make sure we have no regression.

**Comment by thim81** [comment]
@nicklloyd, I might need your assistance on this issue.
In `src/application/globals/injectEnvVariables.ts` we inject the variables and add a "baseUrl"

I'm not sure how to handle this properly.

**Comment by thim81** [comment]
hi @JaredAAT, We have a [PR](https://github.com/apideck-libraries/portman/pull/449) in the making to fix this behavior and handle the baseUrl better.

**Comment by JaredAAT** [comment]
Can't wait to start using this library in anger.

**Comment by thim81** [comment]
hi @JaredAAT, The PR is merged, so the fix will be part of the next release.

**Comment by thim81** [comment]
hi @JaredAAT 

🎊 We just released Portman 1.21.0, which includes the latest version 4.9.0 of openapi-to-postman plus the fix for the variables is included in the release.
